### ISSUE TITLE
[title] locally installed client has an error on Applicaion menu

### ISSUE THREAD

**Issue by Hoseong-Seo** [awslabs/aws-saas-boost#499]
When client/web is installed locally, it has an error.
I start the client with commands "yarn start" and login, 
and then click Application of side menu.
It shows 
ERROR 
Cannot read properties of undefined (reading 'filter')
TypeError: Cannot read properties of undefined (reading 'filter')
    at children (http://localhost:3000/src_settings_ApplicationContainer_js.ecc6ae88e3430a383d74.hot-update.js:431:54)

I think it is related with 419 line of "src/settings/ApplicationComponent.js" 

<Form>
                  <AppSettingsSubform
                    isLocked={hasTenants}
                    certOptions={certOptions}
                    acmConsoleLink={acmConsoleLink}
                    **hostedZoneOptions={hostedZoneOptions.filter(option => option.name.startsWith(formik.values.domainName))}**
                    route53ConsoleLink={route53ConsoleLink}
                  ></AppSettingsSubform>

### Environment

  - **AWS Region  :**
  - **AWS SaaS Boost Version :**
  - **Workload OS  (Linux or Windows) :**

### Other

--- 

This is :bug: Bug Report

**Comment by Hoseong-Seo** [comment]
1. Start the client with commands "yarn start" in local environment(Laptop...)
 2. login in admin  
 3. Click "Application"  menu of left side.
 4. And show error message.

**Comment by PoeppingT** [comment]
@Hoseong-Seo have you configured the `.env` file in your local environment? There is a script in the `client/web` folder named `update_local_env.sh` that you can use to populate your local environment variables for running against your installed SaaS Boost environment.

**Comment by Hoseong-Seo** [comment]
Yes, I did it. If it's not configured, I can not login to Admin Page. I think It can be timing issue(bug).  When it loads application menu, there is no "hostedZoneOptions" value.
<img width="1603" alt="image" src="https://user-images.githubusercontent.com/28926635/231723230-2b429fd4-90a1-4821-b62c-30f7a5ff24ff.png">
